saplings/circuits/src/components/forms/propose_circuit/service.js

```diff
@@ -17,11 +17,11 @@
 import React, { useState, useReducer, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { Service, generateID } from '../data/circuits';
-import { Node } from '../data/nodeRegistry';
-import { PlusButton, MinusButton } from './PlusMinusButton';
+import { Service, generateID } from 'App/data/circuits';
+import { Node } from 'App/data/nodeRegistry';
+import { PlusButton, MinusButton } from 'App/components/PlusMinusButton';
 
-import './ServiceCard.scss';
+import './service.scss';
```

Comment:
NodeDisplay should probably get moved out (it can probably be used in more places), but the ServiceTable is part of the service module.

Part of this re-layout is to provide better organization for where shared components should reside.  The current organization is slapdash.  As part of an upcoming style/minor functional tweaks will probably include more reorganization.

Reply:
Actually, it's a private type - it's reused in the service form and the service table.